### ISSUE TITLE
Make a poster for the Wolf Rayet conference in June

### ISSUE THREAD

**Issue by will-henney** [will-henney/globule-seminario#31]
First, we need to check that there will be posters

[Conference website](https://wruniverse2023.irya.unam.mx/web/)

**Comment by will-henney** [comment]
Jesús says that we can submit a poster.  So the steps will be:
- [ ] Write abstract (@will-henney)
- [ ] Register poster with conference (@RobeReyes)
- [ ] Design poster (@will-henney @RobeReyes)
- [ ] Print popster (@RobeReyes)
- [ ] Attend conference (@RobeReyes)

**Comment by will-henney** [comment]
Ideas for elements to include in the poster:

## Empirical

- [ ] Images of nebula, HST, JWST
- [ ] Close-up images of typical globules
- [ ] Distribution in PA and projected radius
- [ ] Examples of fits to surface brightness profiles
- [ ] Methodology for determining stagnation pressure from H alpha brightness

## Theoretical 

- [ ] Cartoon of the two wind interaction. We could base this on Figure 2 of [Tarango & Henney (2018)](https://ui.adsabs.harvard.edu/abs/2018MNRAS.477.2431T)
- [ ] Equations for photoevaporation flow and for pressure balance between the two streams. 
- [ ] Relationship to stellar wind properties Mdot and V infinity
- [ ] Effect of viewing angle inclination 

## Results

- [ ] Graph of pressure versus projected distance, showing theoretical curves for stellar wind ram pressure
- [ ] Interpretation in terms of projection effect for the inner globules. Deduced distribution of true separations
- [ ] Fact that outer globule shells have pressure higher than expected for a free-flowing wind. Back reaction of interaction with inner globules upon the stellar wind

**Comment by RobeReyes** [comment]
During the conference there was a dynamic where we had to present our poster to everyone in just a couple of minutes. Our poster was presented on Wednesday, but it remained there for the rest of the week. Not only was it presented on that occasion, but I also took advantage of the coffee break to explain it to others personally and in more detail. Even some people asked me about it during dinner and other meals where we socialized.
Unfortunately, it was not much of their interest for the conference attendees, but it was still a good practice to present it to other people.
They also pointed out to me that some phrases in the abstract were not very easy to understand, so I needed to explain that in more detail.

![IMG_20230623_204449](https://github.com/will-henney/globule-seminario/assets/123852273/d675c79e-7374-4a71-96b2-25b712f1646e)

It was a great and enjoyable experience!!!
